plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadata.java

```diff
@@ -517,7 +517,7 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         SchemaTableName schemaTableName = handle.isNamedRelation()
                 ? handle.getRequiredNamedRelation().getSchemaTableName()
                 // TODO (https://github.com/trinodb/trino/issues/6694) SchemaTableName should not be required for synthetic ConnectorTableHandle
-                : new SchemaTableName("_prepared", "query");
+                : new SchemaTableName("_generated", "_generated_query");
```

Comment:
could we chain some operations here too, e.g: `_generated_query_agg_join_pushdown`?

Reply:
We don't know them at this point, and it feels redundant to track them just for that purpose. 
note that in EXPLAIN we also print the actual query.

Reply:
> note that in EXPLAIN we also print the actual query.

what do you mean by actual query?

Reply:
when `! handle.isNamedRelation()` then this handle is based on a query, build from subsequent pushdowns. this query is visible in EXPLAIN, as part of TH toString